src/python/Utils/MathUtils.py

```diff
@@ -15,11 +16,11 @@ def quantize(inputVal, quanta):
     """
     if isinstance(inputVal, basestring):
         inputVal = float(inputVal)
-    elif not isinstance(inputVal, (int, float, long)):
+    elif not isinstance(inputVal, (int, float)):
         msg = "Input value has to be either int or float, not %s" % (type(inputVal))
         raise ValueError(msg)
 
-    if isinstance(quanta, (basestring, float, long)):
+    if isinstance(quanta, (basestring, float, int)):
```

Comment:
I'm afraid this might break something. Many of the MySQL queries return long data.

Reply:
This is strange.

In py3 the int type is like the py2 long type. Futurize let us import the backported py3 int type to py2, so that using int in py2 with futurize import is like using long type in py2 without futurize.

I do not know why this import is not done by default by futirize.

References:
* http://python-future.org/compatible_idioms.html#long-integers
* http://python-future.org/what_else.html#int